.github/kokoro/xc7.sh

```diff
@@ -30,4 +30,17 @@ echo "----------------------------------------"
 	popd
 )
 echo "----------------------------------------"
+
+echo
+echo "========================================"
+echo "Running installed toolchain tests"
+echo "----------------------------------------"
+(
+	sudo pip install git+https://github.com/SymbiFlow/fasm.git
```

Comment:
There is never a justification for using ``sudo`` here.  Either depend on installing it to the local python cache, or make an explicit python virtualenv

Reply:
fixed